src/Uno.UI/MixinGeneration.targets

```diff
@@ -122,7 +122,7 @@
 	<Message Text="Generating mixins" Importance="high" />
 
 	<!-- This allows for the mixins to generate in parallel -->
-	<MSBuild BuildInParallel="true" Projects="@(MixinDefinition)" StopOnFirstFailure="True" Targets="_InnerGenerateMixins" />
+	<MSBuild BuildInParallel="false" Projects="@(MixinDefinition)" StopOnFirstFailure="True" Targets="_InnerGenerateMixins" />
```

Comment:
Why do you need to remove it ?

Reply:
Removing temporarily.

I think we are hitting a race fixed in https://github.com/mono/t4/pull/101.

The PR on mono/t4 was merged in March 2021, and the latest version on NuGet.org is released in February 2021.

I want to get things to pass first, then I'll see how I can consume a more recent release.